tools/grab_starred_links.py

```
@@ -220,21 +215,23 @@
-    for entry in feed.entries:
-        
-        if create_hugo_post(entry):
-            new_entries_count += 1
-    
-    print(f"Processed {new_entries_count} new entries")
+        new_entries_count = 0
+        for entry in feed.entries:
+            if create_hugo_post(entry):
+                new_entries_count += 1
+
+        logging.info(f"Processed {new_entries_count} new entries")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)